Client/game_sa/CPedSA.cpp

```diff
@@ -98,9 +98,18 @@ void CPedSA::Init()
 
 void CPedSA::SetModelIndex(DWORD dwModelIndex)
 {
-    DWORD dwFunction = FUNC_SetModelIndex;
+    // Delete any existing RwObject first
+    DWORD dwFunction = GetPedInterface()->vtbl->DeleteRwObject;
```

Comment:
This method is declared for 'CEntitySAInterface' already

Reply:
Do you mean DeleteRwObject?

Reply:
Yes

Reply:
Okay. But I still don't understand what you're trying to say.